docs/sources/flow/getting-started/migrating-from-operator.md

```diff
@@ -0,0 +1,262 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/getting-started/migrating-from-operator/
+description: Migrating from Grafana Agent Operator to Grafana Agent Flow
+menuTitle: Migrate from Operator
+title: Migrating from Grafana Agent Operator to Grafana Agent Flow
+weight: 320
+---
+
+# Migrating from Grafana Agent Operator to Grafana Agent Flow
+
+With the release of flow, Grafana Agent Operator is no longer the recommended way to deploy Grafana Agent in Kubernetes. Some of the Operator functionality has been moved into Grafana Agent
+itself, and the remaining functionality has been replaced by our Helm Chart.
+
+- The Monitor types (`PodMonitor`, `ServiceMonitor`, `Probe`, and `LogsInstance`) are all supported natively by Grafana Agent in Flow mode. You no longer are 
+required to use the Operator to consume those CRDs for dynamic monitoring in your cluster.
+- The parts of the Operator that deploy the agent itself (`GrafanaAgent`, `MetricsInstance`, and `LogsInstance` CRDs) are deprecated. We now recommend
+operator users use the [Grafana Agent Helm Chart](https://grafana.com/docs/agent/latest/flow/setup/install/kubernetes/) to deploy the Agent directly to your clusters.
+
+This guide will provide some steps to get started with Grafana Agent for users coming from Grafana Agent Operator.
+
+## Deploy Grafana Agent with Helm
+
+1. You will need to create a `values.yaml` file, which contains options for how to deploy your agent. You may start with the [default values](https://github.com/grafana/agent/blob/main/operations/helm/charts/grafana-agent/values.yaml) and customize as you see fit, or start with this snippet, which should be a good starting point for what the operator does:
+
+    ```yaml
+    agent:
+      mode: 'flow'
+      configMap:
+        create: true
+      clustering:
+        enabled: true
+    controller:
+      type: 'statefulset'
+    replicas: 2
+    crds:
+      create: false
+    ```
+
+    This config will deploy Grafana Agent as a `StatefulSet` using the built in [clustering](https://grafana.com/docs/agent/latest/flow/concepts/clustering/) functionality to allow distributing scrapes across all Agent Pods. 
+    
+    This is not the only deployment mode possible. For example, you may desire a `DaemonSet` if collecting host level logs or metrics. See [this page](https://grafana.com/docs/agent/latest/flow/setup/deploy-agent/) for more details about different topologies.
+
+2. Create a flow config file, `agent.river`.
+
+    You can add any config you need directly to this file.
```

Comment:
Might be good to link to the guide for migrating static configs to flow ones, e.g.:
```suggestion
    You can add any config you need directly to this file. See [Migrating from Static](https://grafana.com/docs/agent/latest/flow/getting-started/migrating-from-static/) for how to migrate a static mode config to Flow.
```

Reply:
Should we also add an example of agent flow config? If in the first step were are showing an snippet with an specific config example, why not here?

Reply:
If they're operator users, they won't have directly interacted with static config, so I'm not sure how relevant that is. Since all the config comes from crds, there isn't any real "initial config", other than what they add below to correspond with a metrics instance. Maybe I can clarify that at least.